src/core/services/theming/theming.spec.js

```diff
@@ -785,6 +797,40 @@ describe('md-theme directive', function() {
       expect($log.warn.calls.count()).toBe(0);
     });
   });
+
+  it('should accept string as a theme', function() {
+    inject(function($compile, $rootScope, $mdTheming) {
```

Comment:
More inconsistency here. I'm not going to argue about the inject part, but please be consistent with this vs line 773. I don't really care which you pick as long as the whole file is the same.